### ISSUE TITLE
Problem with the Sigmamodel 

### ISSUE THREAD

**Issue by asgerkromand** [medialab/ipysigma#196]
After upgrading to 0.22.0 I have experencied problems with the ipywidget. I'm using version 8.0.3 of ipywidgets and my jupyterlab version is 3.2.1. It gives me this error message (text below photo). I also save the networks to a html-file, and the networking do not have problems being displayed there:

<img width="990" alt="billede" src="https://user-images.githubusercontent.com/61695016/207319568-8e4faa60-6b4e-42ff-a184-184f5316d38d.png":

[Open Browser Console for more detailed log - Double click to close this message]
Failed to load model class 'SigmaModel' from module 'ipysigma'
loadClass@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/134.78223085640d026773a5.js?v=78223085640d026773a5:1:74977
loadModelClass@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.b0e841b75317744a7595.js?v=b0e841b75317744a7595:1:10729
_make_model@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.b0e841b75317744a7595.js?v=b0e841b75317744a7595:1:7517
new_model@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.b0e841b75317744a7595.js?v=b0e841b75317744a7595:1:5137
handle_comm_open@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.b0e841b75317744a7595.js?v=b0e841b75317744a7595:1:3894
134/v/this._handleCommOpen@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/134.78223085640d026773a5.js?v=78223085640d026773a5:1:73393
_handleCommOpen@http://localhost:8888/static/lab/jlab_core.075245c92eb560303156.js?v=075245c92eb560303156:2:938550

**Comment by Yomguithereal** [comment]
I am not well equipped to find solutions for this kind of things. Usually I try reloading jupyter then uninstall/reinstall of the lib, jupyter or else because I don't understand how their stuff is often failing to deal with upgrade and I don't think I can do anything on my end to fix it unfortunately. Here it seems your notebook is unable to get the necessary javascript code from the kernel server typically.

If you are using jupyter lab you can also use the command `jupyter labextension list` and see if everything looks fine?

**Comment by asgerkromand** [comment]
That's fair enough. Thanks for the suggestion of the `jupyter labext. list`. They are all `enabled ok`
 I've tried to google my way around the problem, and it seems to be a general problem for ipywdiget. 

The only solution I've found so far is downgrading ipysigma to 0.21.0 if I want the wdiget to display my network. But as I'm in general using the hmtl-files, I guess it is not a serious problem for me in the end.

Thank u for reply.
